From 3ec947a9717509e204c7f09609717951beb7ef04 Mon Sep 17 00:00:00 2001
From: Hans de Goede <hdegoede@redhat.com>
Date: Mon, 17 Feb 2014 17:11:58 +0100
Subject: [PATCH] sunxi-mmc: Stop claiming UHS modes

UHS requires a regulator to set the signaling voltage, as well as special
code to handle the signal voltage switching. We've never added the code
for this from the android driver to the upstream driver, so stop claiming
support for this.

Signed-off-by: Hans de Goede <hdegoede@redhat.com>
---
 drivers/mmc/host/sunxi-mmc.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/drivers/mmc/host/sunxi-mmc.c b/drivers/mmc/host/sunxi-mmc.c
index c1a9d8a..2eaed10 100644
--- a/drivers/mmc/host/sunxi-mmc.c
+++ b/drivers/mmc/host/sunxi-mmc.c
@@ -803,8 +803,7 @@ static int sunxi_mmc_probe(struct platform_device *pdev)
 		mmc->ocr_avail = MMC_VDD_32_33 | MMC_VDD_33_34;
 
 	mmc->caps = MMC_CAP_MMC_HIGHSPEED | MMC_CAP_SD_HIGHSPEED |
-		MMC_CAP_UHS_SDR12 | MMC_CAP_UHS_SDR25 | MMC_CAP_UHS_SDR50 |
-		MMC_CAP_UHS_DDR50 | MMC_CAP_SDIO_IRQ | MMC_CAP_DRIVER_TYPE_A;
+		     MMC_CAP_SDIO_IRQ;
 	mmc->caps2 = MMC_CAP2_NO_PRESCAN_POWERUP;
 
 	ret = mmc_add_host(mmc);
-- 
1.8.5.5

